build-scripts/build.sh

```diff
@@ -8,7 +8,7 @@ num_jobs=3
 
 function run_tests
 {
-    $WINE "$@" -d yes --rng-seed time $EXTRA_TEST_OPTS
+    $WINE "$@" -d yes --rng-seed time $EXTRA_TEST_OPTS | grep -v "0.0[0-9]{2} s:"
```

Comment:
```suggestion
    $WINE "$@" -d yes --use-colour yes --rng-seed time $EXTRA_TEST_OPTS | grep -Ev "^0\.0[0-9]{2} s:"
```

Reply:
Piping through something means colour is off by default, so we need to turn that on again.  I've slightly corrected your `grep` syntax and made it less likely to have false-positives.

You also need to add a `set -o pipefail` somewhere in the script (possibly at the top; possibly just in this function) otherwise failing tests won't actually cause the tests to fail.  The default shell behaviour is to ignore the return codes from all but the last command in a pipeline.